dev-packages/native-webpack-plugin/src/native-webpack-plugin.ts

```diff
@@ -58,15 +58,11 @@ export class NativeWebpackPlugin {
         let ripgrepIssuer: string | undefined;
         compiler.hooks.initialize.tap(NativeWebpackPlugin.name, async () => {
             const directory = path.resolve(compiler.outputPath, 'native-webpack-plugin');
-            if (fs.existsSync(directory)) {
-                await fs.promises.rm(directory, {
-                    force: true,
+            if (!fs.existsSync(directory)) {
+                await fs.promises.mkdir(directory, {
                     recursive: true
                 });
             }
```

Comment:
```suggestion
            await fs.promises.mkdir(directory, { recursive: true });
```

What do you think? Isn't it the same and easier to read? It's better not to rely on sync checks and then dispatch some logic.

Reply:
You're right. I forgot that `mkdir` is a no-op in case the directory already exists :)